ocs_ci/ocs/ocp.py

```
@@ -1157,14 +1162,14 @@ def wait_for_resource(
                 elif sample.get("kind") == "List":
                     in_condition = []
                     in_condition_len = 0
-                    actual_status = []
+                    current_status = []
                     sample = sample["items"]
```

Ruff (0.16.1)

[warning] `for` loop variable `sample` overwritten by assignment target

(PLW2901)